stock_quant_manual_assign/__openerp__.py

```diff
@@ -25,5 +25,5 @@
         "wizard/assign_manual_quants_view.xml",
         "views/stock_move_view.xml",
     ],
-    "installable": False,
```

Comment:
in this file, you should remove the contributor (they should be in the README.rst